specification/sections/01-flag-evaluation.md

```diff
@@ -119,32 +161,63 @@ MyStruct myStruct = client.getObjectValue<MyStruct>('structured-flag', { text: '
 
 See [evaluation context](./03-evaluation-context.md) for details.
 
-##### Condition 1.3.2
+#### Condition 1.3.3
 
 > The implementation language differentiates between floating-point numbers and integers.
 
-###### Conditional Requirement 1.3.2.1
+##### Conditional Requirement 1.3.3.1
 
 > The client **SHOULD** provide functions for floating-point numbers and integers, consistent with language idioms.
 
-```go
-// example in GO
-GetIntValue(flag string, defaultValue int64, evalCtx EvaluationContext, options ...EvaluationOption) (int64, error)
+```
+int getIntValue(String flag, int defaultValue);
 
-GetFloatValue(flag string, defaultValue float64, evalCtx EvaluationContext, options ...EvaluationOption) (float64, error)
+long getFloatValue(String flag, long defaultValue);
 ```
 
 See [types](../types.md) for details.
 
-##### Requirement 1.3.3
+#### Requirement 1.3.4
 
 > The `client` **SHOULD** guarantee the returned value of any typed flag evaluation method is of the expected type. If the value returned by the underlying provider implementation does not match the expected type, it's to be considered abnormal execution, and the supplied `default value` should be returned.
 
-#### 1.4. Detailed Flag Evaluation
+### 1.4. Detailed Flag Evaluation
 
 [![hardening](https://img.shields.io/static/v1?label=Status&message=hardening&color=yellow)](https://github.com/open-feature/spec/tree/main/specification#hardening)
 
-##### Requirement 1.4.1
+
+#### Condition 1.4.1
+
+[![experimental](https://img.shields.io/static/v1?label=Status&message=experimental&color=orange)](https://github.com/open-feature/spec/tree/main/specification#experimental)
+
+> The implementation uses the single-context paradigm.
+
+see: [single-context paradigm](../glossary.md#single-context-paradigm)
+
+##### Conditional Requirement 1.4.1.1
+
+> The `client` **MUST** provide methods for detailed flag value evaluation with parameters `flag key` (string, required), `default value` (boolean | number | string | structure, required), and `evaluation options` (optional), which returns an `evaluation details` structure.
+
+```typescript
```

Comment:
Not really, it's kinda nothing.

If you don't put _some_ language though, you dont get syntax highlighting, which looks... not good.

I'm considering converting most examples to Java because it's very readable, and pretty commonly understood.